package.json

```diff
@@ -50,7 +50,7 @@
     "eslint-plugin-import": "2.14.0",
     "eslint-plugin-jsx-a11y": "6.1.1",
     "eslint-plugin-react": "7.11.1",
-    "evergreen.js": "^1.2.1",
+    "evergreen.js": "^1.2.2",
```

Comment:
This will change the checksum of the package, so be sure to npm install and commit the changes to package-lock.json